manifest.yml

```diff
@@ -0,0 +1,7 @@
+# Cloud.gov sandbox config. See readme for setup.
+applications:
+- name: usds-website-staging
```

Comment:
While it's a staging environment we have in mind in this PR, I'm wondering about someone who might try to re-use these instructions to deploy to a production environment. My concern: in this file we hard-code the environment in the application name (`usds-website-staging`) - should we consider making this file more explicitly staging-specific (alter the filename?) or use an environment variable for the application name? Or: out of scope for this PR?

Reply:
It would definitely be cleaner and more forward thinking to have the name configured by an environment variable. 

I think the extra steps to deploy a production site this way are enough that no one should be trying to just do so based on what we implemented so far (i.e. it's in a sandbox without a proper cname configured or ssl certificates etc.). With just Drew as sole owner for now I think a cleaner implementation can wait until it's needed but then will there be the space to do it right?

Reply:
I changed the site name to `website-staging` to be more generic and updated the comment on manifest.yml to reflect it is not presently intended for production.

Definitely interested in the more advanced setup in the future though.

Reply:
Thanks for the feedback! The current state looks reasonable to me, and I'm on board with the adjustments that have been made. Ship it!